english/adipose/adipose.md

```diff
@@ -4,9 +4,9 @@ You die from fat drain.
 
 You are absorbed into the marshmallow wall and now you wait with the others for new recruits.
 
-You emerge from the wall a marshmallow human. YOu now resemble the michelin man. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.